lib/Common/Memory/RecyclerPointers.h

```diff
@@ -515,13 +520,23 @@ void qsort_s(T* arr, size_t count, const Comparer& comparer, void* context)
     typedef typename AllocatorWriteBarrierPolicy<Allocator, ItemPolicy>::Policy Policy;
     _QuickSortImpl<Policy>::Sort(arr, count, comparer, context);
 }
+
+#ifndef _WIN32
+template<class T, class PolicyType = T, class Allocator = Recycler, class Comparer>
+void qsort_s(T* arr, size_t count, size_t size, const Comparer& comparer, void* context)
+{
+    typedef typename _ArrayItemWriteBarrierPolicy<PolicyType>::Policy ItemPolicy;
+    typedef typename AllocatorWriteBarrierPolicy<Allocator, ItemPolicy>::Policy Policy;
+    _QuickSortImpl<Policy>::Sort(arr, count, comparer, context, size);
+}
+#else
```

Comment:
Once again, we use custom qsort on xplat. Instead of implementing qsort on 2 different places, combined all in here.